app/views/cookies/consent.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="row">
+  <div class="col-md-8">
+    <h1>Cookie Consent</h1>
+
+    <%= render partial: 'general/cookies_info' %>
+
+    <% analytics_txt = "Google Analytics" if (Seek::Config.google_analytics_enabled && !Seek::Config.google_analytics_enabled) %>
+    <% analytics_txt = "Matomo" if (Seek::Config.piwik_analytics_enabled && !Seek::Config.google_analytics_enabled) %>
+    <% analytics_txt = "Google Analytics and Matomo" if (Seek::Config.piwik_analytics_enabled && Seek::Config.google_analytics_enabled) %>
+    <% analytics_txt = "Google Analytics" if (Seek::Config.google_analytics_enabled) %>
```

Comment:
The first option can never be satisfied 

Could probably simplify this by adding the strings to an array and calling `.join(' and ')`

Reply:
Did that, but also had to change 2 different parts below (tracking shouldn't be shown if not activated, the last lines were redundant and kind of stupid). Not sure there isn't a better way to show the selected choices.